compiler/testData/diagnostics/tests/enum/constructorWithDefaultParametersOnly.fir.kt

```diff
@@ -18,4 +18,4 @@ enum class TestMultipleConstructors(val x: String = "", val y: Int = 0) {
 enum class TestVarargs(val x: Int) {
     TEST;
     constructor(vararg xs: Any) : this(xs.size)
-}
\ No newline at end of file
+}
```

Comment:
Nit: perhaps you can ignore this comment, but technically, no test data are changed by this commit, right?

I assume you played around (and those were changed by formatting) and then found where to change w.r.t. PSI consistency check?

Reply:
Welp, yeah something like that. Let me revert this file.

Reply:
Well, I didn't mean only this file, but many test changes in the 1st commit.

Reply:
Ohhh I didn't realize these many files were touched!